release.json

```diff
@@ -24,7 +24,7 @@
         "WINDOWS_APMINJECT_VERSION": "1.1.3",
         "WINDOWS_APMINJECT_SHASUM": "5fdd62a84e640204386b9c28dc2e3ac5d9b8adde6427cb9f5914619f94d7b5bd"
     },
-    "nightly-a7": {
+    "a7": {
```

Comment:
I can't comment directly on the line, but can you keep `SECURITY_AGENT_POLICIES_VERSION` pointing to an actual tag ? This versioning is useful for us, and we can move to master at a later date

Reply:
It's important especially in the case of multiple releases happening at the same time (we wouldn't want 7.70.1 bundling the same policies as 7.71.0 for example)

Reply:
I can do that; just note that that means that it will become pinned in regular pipelines too. I don't know what your current process is for managing this dependency, other similar dependencies (e.g. integrations-core) use tags that match those on the agent (e.g. 7.56.0-rc.2).

Reply:
Currently what we do is `v.0.xx.0` matches 7.xx.0, and if we need to update the policy we will create .1 or .2 etc

Reply:
👍 I just pushed the change.